packages/redux/src/useReduxDevTools.ts

```diff
@@ -7,18 +7,29 @@ import { Store } from '@reduxjs/toolkit';
  *
  * @param store - The Redux store.
  */
+
+let lastAction = null;
 export function useReduxDevTools(store: Store) {
   const client = useDevToolsPluginClient('redux');
 
   useEffect(() => {
-    client?.sendMessage("storeUpdated", store.getState());
+    client?.sendMessage('storeUpdated', { store: store.getState(), lastAction });
+
     const unsubscribeFn = store.subscribe(() => {
       const state = store.getState();
-      client?.sendMessage("storeUpdated", state);
+      client?.sendMessage('storeUpdated', state);
```

Comment:
```suggestion
      client?.sendMessage('storeUpdated', { store: store.getState(), lastAction });
```

should we have the consistent `storeUpdated` payload?